lib/slack/client.ex

```diff
@@ -0,0 +1,257 @@
+defmodule Slack.Client do
+  @moduledoc """
+  A datastructure that allows you to monitor, inspect, and react to the state of
+  your Slack team.
+  """
+  
+  if :lt == Version.compare(System.version, "1.1.0") do
+    
+    @derive [Access]
+    
+  else
+  
+    @behaviour Access
+    
+    @type key :: any
+    @type value :: any
+
+    @doc """
```

Comment:
Not sure if we need to document this, or `get_and_update` since it's strictly for backwards compat